src/gpuarray_elemwise.c

```diff
@@ -31,6 +31,7 @@ struct _GpuElemwise {
 STATIC_ASSERT(GEN_CONVERT_F16 == GE_CONVERT_F16, same_flags_value_elem1);
 
 #define is_array(a) (ISCLR((a).flags, GE_SCALAR))
+#define is_rw_array(a) (ISSET((a).flags, GE_READ) && ISSET((a).flags, GE_WRITE))
```

Comment:
No, that should not be allowed, since it doesn't really make sense.  We have 5 output values to store in a single memory location.  How do we decide which do we store?  Do we take the mean? The sum?